### ISSUE TITLE
Twitter and Twitch verified pubs are being added to a-c table

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#6142]
## Description 

Twitter and Twitch verified pubs are being added to a-c table

## Steps to Reproduce

   1. Brave 0.69.128
   2. Visit verified twitch pubs (https://www.twitch.tv/jennrhim/videos)
   3. Watch any video and spend min time
   4. Visit verified twitter pub (https://twitter.com/kamiljozwiak)
   5. Twitter and twitch pubs are not added to a-c table

## Actual result:

Twitter and Twitch verified pubs are being added to a-c table
![image](https://user-images.githubusercontent.com/38657976/65503773-3b93df80-dee3-11e9-84d1-fe1334c1715c.png)

## Expected result:

Verified pubs should be added to a-c table
## Reproduces how often: 

Always

## Brave version (brave://version info)

Brave | 0.68.140 Chromium: 77.0.3865.90 (Official Build) (64-bit)
-- | --
Revision | 58c425ba843df2918d9d4b409331972646c393dd-refs/branch-heads/3865@{#830}
OS | Windows 10 OS Version 1803 (Build 17134.1006)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @NejcZdovc

**Comment by GeetaSarvadnya** [comment]
The same issue for Reddit pubs as well in the panel pubs name is displayed but pubs is not getting added to a-c table, the only domain name is added 

![image](https://user-images.githubusercontent.com/38657976/65505898-f0c89680-dee7-11e9-8e8d-9d54f3b1b545.png)
![image](https://user-images.githubusercontent.com/38657976/65505922-fd4cef00-dee7-11e9-9a95-9b1815418b85.png)

**Comment by NejcZdovc** [comment]
we don't have yet support for AC for Twitter

**Comment by NejcZdovc** [comment]
we don't have support for reddit AC

**Comment by bsclifton** [comment]
@GeetaSarvadnya is this supposed to be: `Twitter and Twitch verified pubs are *NOT* being added to a-c table`?

**Comment by GeetaSarvadnya** [comment]
@bsclifton Yup, you're right it should `Twitter and Twitch verified pubs are *NOT* being added to a-c table?` - I have updated the description. Looks like its regression for Twitch, `0.68.131 Chromium: 76.0.3809.100` onwards this stopped working.

- `0.68.140 Chromium: 77.0.3865.90` - Twitch verified pubs are NOT being added to a-c table
- `0.68.139 Chromium: 77.0.3865.90` - Twitch verified pubs are NOT being added to a-c table
- `0.68.138 Chromium: 77.0.3865.75`  - Twitch verified pubs are NOT being added to a-c table
- `0.68.132 Chromium: 76.0.3809.132`- Twitch verified pubs are NOT being added to a-c table
-  `0.68.131 Chromium: 76.0.3809.100` -Twitch verified pubs are being added to a-c table
- `0.68.129 Chromium: 76.0.3809.100` - Twitch verified pubs are being added to a-c table 

![image](https://user-images.githubusercontent.com/38657976/65570725-b4e11000-df7f-11e9-95a6-e4fca1576ddc.png)

**Comment by LaurenWags** [comment]
I've noticed this happens sometimes, but not others for me. Sometimes  twitch publishers are added immediately and other times it can take up to 5 minutes. This wasn't always the case though, so something seems like it's wrong. I believe @kjozwiak has encountered this as well.

**Comment by LaurenWags** [comment]
Still encountering this with 

Brave | 1.11.65 Chromium: 83.0.4103.97 (Official Build) dev (64-bit)
-- | --
Revision | 326d148b9655369b86498d9ecca39f63dd2bdd2d-refs/branch-heads/4103@{#657}
OS | macOS Version 10.14.6 (Build 18G3020)

- Confirmed while testing https://github.com/brave/brave-browser/issues/9829

**Comment by LaurenWags** [comment]
Noticed with 1.19.x that after a browser restart twitch publishers are more reliably being added to AC, but without the restart they are not added.

```
Brave | 1.19.75 Chromium: 87.0.4280.101 (Official Build) dev (x86_64)
-- | --
Revision | 9407c80213cda69c2b7abcb4fa8e3f74488f4956-refs/branch-heads/4280@{#1807}
OS | macOS Version 10.15.7 (Build 19H15)
```

**Comment by GeetaSarvadnya** [comment]
Seeing the same issue in Windows 10 x64 - Twitch pubs are added to AC table only after browser restart

```

Brave | 1.19.75 Chromium: 87.0.4280.101 (Official Build) dev (64-bit)
-- | --
Revision | 9407c80213cda69c2b7abcb4fa8e3f74488f4956-refs/branch-heads/4280@{#1807}
OS | Windows 10 OS Version 2004 (Build 19041.685)

```
![image](https://user-images.githubusercontent.com/38657976/103763119-66a23400-503f-11eb-8e6c-448dfd864b0e.png)

**Comment by LaurenWags** [comment]
Logged https://github.com/brave/brave-browser/issues/13391 for restart as noted in https://github.com/brave/brave-browser/issues/6142#issuecomment-754662161 and https://github.com/brave/brave-browser/issues/6142#issuecomment-755243458

**Comment by Miyayes** [comment]
A-C sunsetted.
